docker-compose.yml-example

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.5"
```

Comment:
Is there a special reason for increasing the version? (just curious)

Reply:
The network name entry is introduced in compose version 3.5
https://docs.docker.com/compose/compose-file/#name-1

Reply:
Thanks